src/pwm.rs

```diff
@@ -229,12 +210,12 @@ macro_rules! pwm_all_channels {
 
                 //NOTE(unsafe) atomic write with no side effects
                 fn disable(&mut self) {
-                    unsafe { bb::clear(&(*$TIMX::ptr()).ccer, 0) }
+                    unsafe { (*$TIMX::ptr()).ccer.modify(|_,w| w.cc1e().clear_bit()); }
```

Comment:
Sorry, I missed that before, I used bit-banding here because a channel can race with another one while modifying this register. This is also the case for the other `disable`s/`enable`s you changed here.